src/Components/Forms.jsx

```diff
@@ -0,0 +1,212 @@
+import { TextField } from '@mui/material';
+import MenuItem from '@mui/material/MenuItem';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import Typography from '@mui/material/Typography';
+import Snackbar from '@mui/material/Snackbar';
+import Alert from '@mui/material/Alert';
+import './Forms.scss'
+import axios from 'axios';
+import { useState } from 'react';
+import { useUserDetails } from '../Contexts/Usercontext';
+import { Userinfo } from './Userinfo';
+import { aore, afra, achm } from '../data';
+import 'react-toastify/dist/ReactToastify.css';
+
+
+export const Forms = () => {
+
+    const { User, setUser, checkID, checkSub } = useUserDetails()
+
+    const [error, setError] = useState(false)
+    const [info, setinfo] = useState(false)
+    const [snackopen, setSnackOpen] = useState(false);
+
+    let allmaps = {};
+
+
+    async function handleSubmit(e) {
+        e.preventDefault();
+        console.log('handle');
+        console.log(User)
+
+
+
+        if (User.environment === 'Production') {
+            console.log('Production Environment');
+
+            let userExists = false;
+
+            if (aore.wxUserMapping && aore.wxUserMapping[User.userdetail]) {
+                console.log(aore.wxUserMapping[User.userdetail]);
+
+                const { wxUserId, msUserId, wxOrgId } = aore.wxUserMapping[User.userdetail];
+                return <Userinfo wxUserId={wxUserId} msUserId={msUserId} wxOrgId={wxOrgId} />;
+                
+            }
+            if (achm.wxUserMapping && achm.wxUserMapping[User.userdetail]) {
+                console.log(achm.wxUserMapping[User.userdetail]);
+                const { wxUserId, msUserId, wxOrgId } = achm.wxUserMapping[User.userdetail];
+                return <Userinfo wxUserId={wxUserId} msUserId={msUserId} wxOrgId={wxOrgId} />;
+                
+            }
+            if (afra.wxUserMapping && afra.wxUserMapping[User.userdetail]) {
+                console.log(afra.wxUserMapping[User.userdetail]);
+                const { wxUserId, msUserId, wxOrgId } = afra.wxUserMapping[User.userdetail];
+                return <Userinfo wxUserId={wxUserId} msUserId={msUserId} wxOrgId={wxOrgId} />;
+                
+            }
+
+            // If the user wasn't found in any of the objects, show "not exist" alert
+            if (!userExists) {
+                window.alert('User does not exist');
+            }
+        }
+        else {
+
+
+            const api = `https://msft-presence-service.int-first-general1.ciscospark.com/msft-presence-service/api/v1/allUserMapping`
```

Comment:
Make constants for INT & PROD URL and use the constant instead of using complete URL for code readability perspective.
You can like this:
 INT_URL =  https://msft-presence-service.int-first-general1.ciscospark.com
PROD_URL = depends on data centers like below
PROD-ACHM-Url : https://msft-presence-service-a.wbx2.com/
PROD-AFRA-Url : https://msft-presence-service-k.wbx2.com/
PROD-AORE-Url : https://msft-presence-service-r.wbx2.com 

Then form api value as const api = INT_URL + "the rest of the URL parameters";
All the strings used in the API URL can be made as constants like SERVICE = "service", API = "api",  V1 = "v1"